### ISSUE TITLE
Merge into diff_cover

### ISSUE THREAD

**Issue by DominicOram** [dperl-dls/pyright_diff_quality_plugin#2]
We should try to avoid having to maintain this ourselves and merge it with the wider community. As such we should do https://github.com/Bachmann1234/diff_cover/issues/379

**Comment by d-perl** [comment]
Yes we should, but it'll be a little more involved to add support for all the various things one can specify with arguments and format it to the internal structure of diff_cover.

I'll probably look at this next week.
